components/layout/index.jade

```diff
@@ -1,53 +1,41 @@
-doctype html
 - colorClass = sd.IS_INVERTED ? 'is-inverted' : ''
 - loggedInClass = user ? 'is-logged-in' : 'not-logged-in'
-html(class="#{colorClass} #{loggedInClass}")
-  head
-    title
-      | Are.na
-      if channel
-        |  / #{channel.get('title')}
-      else if author
-        |  / #{author.get('username')}
-
-    include ../../components/meta/templates/meta
-    block meta
-    link( type='text/css', rel='stylesheet', href=asset('/assets/all.css') )
-    block styles
+- extraClasses = ''
 
-    - extraClasses = ''
+doctype html
+html( class="#{colorClass} #{loggedInClass}" )
+  head
+    include modules/title
+    include modules/meta
+    include modules/styles
 
     block vars
 
-  body(class=extraClasses)
+  body( class= extraClasses )
+    //- TODO: Refactor to use modalize, only thing that uses this is AuthView
     #modal-container
       //- Rendered client-side
 
-    #l-lightbox-container
-      //- Rendered client-side
-
```

Comment:
I'm 99% sure nothing uses this. The only thing I can find is part of the block view ganks some styles from that component.